### ISSUE TITLE
document editor - button for save and continue

### ISSUE THREAD

**Issue by ghost** [chamilo/chamilo-lms#2795]
**Is your feature request related to a problem? Please describe.**
When editing a document, sometimes it is usefull to save in between and continue.

**Describe the solution you'd like**
A save and continue button next to the save button, so you can decide if you are done with editing and close or if you want to continue.

**Comment by ywarnier** [comment]
Good idea. There was an idea at some point about doing that automatically, but this was complicated to implement. However, a "Save progress" or "Save and continue" button that simply saves and reopens the edit page could do the trick, for now, indeed.

**Comment by AngelFQC** [comment]
Is related with the #2663?

**Comment by ywarnier** [comment]
Yes, that's the task I was talking about.

**Comment by ywarnier** [comment]
This is practically a duplicate of #2663, so I'm closing it. The auto-save feature is much more practical for everyone. It doesn't require saving anything manually and can be done asynchronously and periodically without changing UI elements
